### ISSUE TITLE
FLUENT UI migration  broken the Grid Layout, UI ICONS, etc

### ISSUE THREAD

**Issue by nsksaisaravana** [SharePoint/sp-dev-docs#5745]
## Category

- [x] Bug

## Describe the bug
SPFX project css was broken due to the recent Flunt ui migation from office ui fabric. I am using the below code to toggle between the mobile and large screens but unfortunately, now it is not working. 

```
<div class="ms-Grid-col ms-sm12 ms-hiddenXxlUp">Visible on smaller screens</div>
<div class="ms-Grid-col ms-sm12 ms-hiddenXlDown">Visible on larger screens</div>
```

## Steps to reproduce
1. Please browse https://developer.microsoft.com/en-us/fluentui#/styles/web/layout
```
<div class="ms-Grid-col ms-sm12 ms-hiddenXxlUp">Visible on smaller screens</div>
<div class="ms-Grid-col ms-sm12 ms-hiddenXlDown">Visible on larger screens</div>
```

## Expected behavior
Above code suppose to toggle between mobile and large screens.

## Environment details (*development & target environment*)

SPFX
REACTJS
Office UI Fabric

### Additional context

I followed all the migration steps but still, it is not working in the production environment. Fluent UI installation steps work only in the development environment but is fails in the production environment.

*Thanks for your contribution! Sharing is caring.*

**Comment by nsksaisaravana** [comment]
Please check the screenshot below, mobile and large screen view are showing. Mobile view suppose to hide but that is not happening.

ms-hiddenXlDown, ms-hiddenXxlUp is not working


![image](https://user-images.githubusercontent.com/17681573/82011368-2957f300-96b8-11ea-828b-74c01eb24e03.png)

**Comment by nsksaisaravana** [comment]
please refer the below tickets as well.

https://github.com/microsoft/fluentui/issues/13112

https://github.com/microsoft/fluentui/issues/13157

**Comment by ecraig12345** [comment]
This is purely an integration issue with your project + SPFx + Fluent UI React + Fabric Core, not an issue with Fluent UI React itself, and it most likely has nothing to do with the Fabric/Fluent UI rename. The issues in the Fluent UI repo have been closed for that reason, but hopefully you can find help here. Please stop commenting on more Fluent UI issues with this problem.

**Comment by andrewconnell** [comment]
Couple things... first, you deleted the context from the issue template which contained context questions we need. Please fix that before we can do anything with this issue. 

Second, there's no context about your project, what packages you're using, etc.

Third, SPFx/SPO hasn't switched anything WRT Office UI Fabric/FluentUI. As @ecraig12345 said, it seems like this is an issue with your project & how you're using/integrating Fabric/FluentUI into it. Providing two lines of HTML isn't enough to go on.

A LOT more context needs to be included before we can move forward with this. Also, please keep in mind this is a forum for SharePoint development issues. Unless there's an issue with SPFx, not sure there's going to be much that can be done here.

**Comment by JackStrap** [comment]
Hi! I experience the same problem in my SPFx project,the ms-Grid-row, ms-Grid-col, ms-lg*, stop working, i did not update any package and it stop working. maybe SharePoint Online change something
Here is my dependencies in my package.json
"dependencies": {
    "@microsoft/sp-core-library": "1.10.0",
    "@microsoft/sp-lodash-subset": "1.10.0",
    "@microsoft/sp-office-ui-fabric-core": "1.10.0",
    "@microsoft/sp-property-pane": "1.10.0",
    "@microsoft/sp-webpart-base": "1.10.0",
    "@pnp/common": "2.0.4",
    "@pnp/graph": "2.0.4",
    "@pnp/logging": "2.0.4",
    "@pnp/odata": "2.0.4",
    "@pnp/sp": "2.0.4",
    "@pnp/spfx-controls-react": "1.17.0",
    "@types/es6-promise": "0.0.33",
    "@types/react": "16.8.8",
    "@types/react-dom": "16.8.3",
    "@types/webpack-env": "1.13.1",
    "datatables.net": "1.10.20",
    "datatables.net-dt": "1.10.20",
    "datatables.net-select": "1.3.1",
    "datatables.net-select-dt": "1.3.1",
    "office-ui-fabric-react": "6.189.2",
    "react": "16.8.5",
    "react-dom": "16.8.5",
    "react-quill": "1.3.5",
    "spfx-navigation": "1.1.0"
  },
Thanks

**Comment by andrewconnell** [comment]
@JackStrap suggest you open a new issue as it's unclear if you're also reporting an SPFx+FluentUI issue...

**Comment by nsksaisaravana** [comment]
@JackStrap 

including following import in root webpart component and it works for me

import '../../../node_modules/office-ui-fabric-core/dist/css/fabric.min.css'

OR

SPComponentLoader.loadCss('https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/11.0.0/css/fabric.min.css'

**Comment by JackStrap** [comment]
@nsksaisaravana, thanks for your response.

You can include in your .scss file in your main container this 
:global{
    @import 'node_modules/office-ui-fabric-react/dist/css/fabric.css';
}
and it will work

**Comment by nsksaisaravana** [comment]
thanks @JackStrap
